lib/cli/index.js

```diff
@@ -5,9 +5,17 @@ var runBinary = require('./run-binary');
 // commands
 var add = require('./cmd-add');
 var help = require('./cmd-help');
+var init = require('./cmd-init');
 
 program.version(require('../../package.json').version);
 
+// donejs new
+program.command('new <folder>')
```

Comment:
For the `new` command, it should be able to create apps, plugins, or generators; not only only new apps. It's essentially the same as add. The word **new** just better articulates the creation effect of commands, which is a convention other CLI tools align with.